packages/hydrogen-react/src/analytics-utils.ts

```diff
@@ -44,9 +44,7 @@ export function parseGid(gid: string | undefined): ShopifyGId {
     return defaultReturn;
   }
 
-  // TODO: add support for parsing query parameters on complex gids
-  // Reference: https://shopify.dev/api/usage/gids
```

Comment:
Please leave this comment in, as we still aren't doing that yet and I would love to have this note here still.